src/Components/Layout/Header/Header.js

```diff
@@ -0,0 +1,71 @@
+import { Box, Button, List, Divider, SwipeableDrawer, Container, IconButton } from '@mui/material';
+import { useState } from 'react';
+import { Link } from 'react-router-dom';
+import MenuIcon from '@mui/icons-material/Menu';
+import logo from '../../../Assets/Logo/logo.png'
+import LoginIcon from '@mui/icons-material/Login';
+import LogoutIcon from '@mui/icons-material/Logout';
+import {useEffect} from 'react';
+import NavLinksList from './NavLinksList';
+
+const Header = () => {
+  const [isOpen, setIsOpen] = useState(false)
+  const isLogged = false;
+  const navLinks = [
+    {text: 'Inicio', link: '/', private:false, active:false},
+    {text: 'Nosotros', link: '/nosotros', private:false, active:false},
+    {text: 'Contacto', link: '/contacto', private:false, active:false},
+    {text: 'Campaña escolar', link: '/school-campaign', private:false, active:false},
+    {text: 'Campaña de juguetes', link: '/toys-campaign', private:false, active:false}
+  ]
+  const activeLink = (pathname) => {
+    navLinks.forEach((navLink, index) => {
+      if(navLink == true)
+        navLinks[index].active = false;
+      else if(navLink.link == pathname)
+        navLinks[index].active = true;
+    })
+  }
```

Comment:
maybe the name of this function should be manageLinkActivation

Reply:
Additionally I think that maybe this could be in a file called HeaderLinks with the links list

Reply:
Changes applied